strawberry/contrib/django/type.py

```diff
@@ -0,0 +1,68 @@
+import strawberry
+from strawberry.contrib.django.converter import convert_django_field
+from strawberry.type import type as strawberry_type
+
+
+def model_type(cls=None, *, model=None, fields=[], is_input=False, is_interface=False):
```

Comment:
do we test is interface? Do we need it? I guess we could check if a model is abstract, if that's what it was meant for :)

Reply:
It could be useful for abstract types. I'll add a test for it.